### ISSUE TITLE
Reduce Firefox IO

### ISSUE THREAD

**Issue by gagarine** [sienori/Tab-Session-Manager#404]
Firefox's standard session manager produce a lot of IO on the disk.

This IO can be fixed by reducing the latency of session write by changing `browser.sessionstore.interval`.

Perhaps this addon can explore if it should disable sessionstore completely. 
Also what information TMS save? I do not want all session data to be stored but only the list of my opentab so I can have a short interval in case of crash but reduce the amount of information saved to the disk.

I'm not familiar with TSM, so perhaps it somethings you already do, but I didn't find any information about it.

The related bug https://bugzilla.mozilla.org/show_bug.cgi?id=1304389

**Comment by sienori** [comment]
Tab Session Manager does not use `browser.sessionstore`.
It uses `browser.tabs.query` to get tab information. https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/query

**Comment by gagarine** [comment]
Yes, I know that. But we can also modify this preferences so installing tab-session-manager will reduce Firefox I0. It was perhaps not clear enough that browser.sessionstore.interval is a preference in about:config.
